asreview/data.py

```diff
@@ -312,6 +313,23 @@ def print_record(self, *args, **kwargs):
         "Print a record to the CLI."
         print(self.format_record(*args, **kwargs))
 
+    @property
+    def match_string(self):
```

Comment:
Private method?

Reply:
Maybe? I think it should be fine as a non-private one, since it may be useful if you want to use a different finding algorithm for example?

Reply:
True. Thanks.